pkgs/dartpad_ui/lib/execution/frame.dart

```diff
@@ -92,7 +92,8 @@ window.onerror = function(message, url, line, column, error) {
       script.writeln('''
 require.config({
   "baseUrl": "$modulesBaseUrl",
-  "waitSeconds": 60
+  "waitSeconds": 60,
+  "onNodeCreated": function(node, config, id, url) { node.setAttribute('crossorigin', 'anonymous'); }
```

Comment:
Can you add a comment here (or perhaps for the `script.writeln` line) with a very brief explanation of what this configuration is for.